CONTRIBUTING.md

```diff
@@ -12,21 +13,54 @@ All forms of contributions are welcomed. Here are some ways to get started with
 - Add new suggestions to the outline
 - Add links to resources .
 
+## Getting Started
+
+- Fork the repository on GitHub.
+- Clone the forked repository, onto your local system:

```

Comment:
There's a little typo
- Clone the forked repository, into your local system:

Reply:
Fixed 👍